frontend/src/components/v2/Alert/Alert.tsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/prop-types */
```

Comment:
had to disable this rule for this component due to some eslint errors

Reply:
Ref: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prop-types.md

Simply stating one of the props you have used is not in types